PluginExamples/Sources/Simple/empty.swift

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can we add a comment to all the empty Swift files saying why we need them to avoid them being deleted by mistake.